src/jrd/os/posix/unix.cpp

```diff
@@ -955,14 +1000,8 @@ static jrd_file* setup_file(Database* dbb,
 		file = FB_NEW_RPT(*dbb->dbb_permanent, file_name.length() + 1) jrd_file();
 		file->fil_desc = desc;
 		file->fil_max_page = MAX_ULONG;
+		file->fil_flags |= flags;
```

Comment:
In winnt.cpp at similar place plain assignment used: `file->fil_flags = flags`
Is it intended difference ?
The final result is the same, I know ;)

Reply:
Just an oversight, fixed.